README.md

```diff
@@ -19,28 +19,53 @@ different technologies.
 Find more information about the client configuration in the README file of the repo:
 [https://github.com/superdesk/superdesk-client-core](https://github.com/superdesk/superdesk-client-core "")
 
-## Installation on fresh Ubuntu 16.04
+## Requirements
+ * MongoDB
+ * ElasticSearch <= 1.7.x
```

Comment:
I was thinking about it, but decided to keep 1.7.
Changed it now, seems we haven't had problems with 2.4, `sd-master.test` works well.
But we still don't have production proof yet :).